samples/tests/mission1-k8s.yaml

```diff
@@ -1,29 +1,84 @@
 drone:
   port: ros #{sitl, ros, cf}
-  params_file: samples/flights/mission1-params.csv #csv file with the same structure as above 
```

Comment:
revert the changes to keep this sample simple. create another sample to demonstrate complidated obstacle types